microk8s-resources/actions/common/utils.sh

```diff
@@ -366,6 +375,26 @@ produce_certs() {
     fi
 }
 
+render_ca_conf() {
+    # Render ca.conf.template to csr.conf.rendered
+
+    local IP_ADDRESSES="$(get_ips)"
+
+    cp ${SNAP_DATA}/certs/ca.conf.template ${SNAP_DATA}/certs/ca.conf.rendered
+    if ! [ "$IP_ADDRESSES" == "127.0.0.1" ] && ! [ "$IP_ADDRESSES" == "none" ]
```

Comment:
This part does not look right. 

The `ca.conf.template` does not have a MOREIPS line. I think we do not need this part.